nncf/parameters.py

```diff
@@ -56,3 +56,17 @@ class DropType(Enum):
 
     ABSOLUTE = "absolute"
     RELATIVE = "relative"
+
+
+@api(canonical_alias="nncf.CompressWeightsMode")
+class CompressWeightsMode(Enum):
+    """
+    Defines a mode for weight compression.
+
+    :param COMPRESSED_INT8: Stands for int8 quantization of all weights.
```

Comment:
```suggestion
    :param COMPRESSED_UINT8: Stands for uint8 quantization of all weights.
```

Reply:
@AlexKoff88 please comment

Reply:
@l-bat, don't think we should mention unsigned here. IMO, it would produce more noise than value.

Reply:
I agree since we are not planning to introduce two options, let's keep the shortest one.